src/index.ts

```diff
@@ -377,36 +360,31 @@ export async function install(
     }
 
     fs.writeFileSync(
-      path.join(destLoc, 'import-map.json'),
+      path.join(config.options.dest, 'import-map.json'),
       JSON.stringify({imports: importMap}, undefined, 2),
       {encoding: 'utf8'},
     );
   }
   Object.entries(assetObject).forEach(([assetName, assetLoc]) => {
-    mkdirp.sync(path.dirname(`${destLoc}/${assetName}`));
-    fs.copyFileSync(assetLoc, `${destLoc}/${assetName}`);
+    mkdirp.sync(path.dirname(`${config.options.dest}/${assetName}`));
+    fs.copyFileSync(assetLoc, `${config.options.dest}/${assetName}`);
   });
   return true;
 }
 
 export async function cli(args: string[]) {
-  const {
-    help,
-    sourceMap,
-    babel = false,
-    exclude = ['**/__tests__/*', '**/*.@(spec|test).@(js|mjs)'],
-    include,
-    nomodule,
-    nomoduleOutput = 'app.nomodule.js',
-    optimize = false,
-    strict = false,
-    clean = false,
-    dest = 'web_modules',
-    externalPackage: externalPackages = [],
```

Comment:
All of these defaults still exist! They were just moved into one central place: `defaultConfig` in `config.ts`.